web_widget_text_markdown/static/src/less/web_widget_text_markdown.less

```diff
@@ -0,0 +1,18 @@
+.dz-preview {
```

Comment:
would be better to isolate the styles by wrapping them in the widget's class

Reply:
bootstrap markdown doesn't have an option to add a custom class :/
These CSS rules are generic, other more specific can overwrite them.

Reply:
I'm talking about the odoo widget class: you should have a specific class for it and the dropzone elements should stay into the widget element, so that you could wrap styles into `.o_widget_text_markdown *` or similar.